oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/IndexUpdate.java

```diff
@@ -82,6 +82,10 @@ public class IndexUpdate implements Editor, PathSource {
     // This counter is cyclically incremented till indexJcrTypeInvalidLogLimiter and then reset to 0
     private static volatile long cyclicExecutionCount = INDEX_JCR_TYPE_INVALID_LOG_LIMITER;
 
+    // Warnings about missing index providers are rate limitted, so that the log file is not filled with them.
```

Comment:
typo + minor suggestion
```suggestion
    // Warnings about missing index providers are rate limited, so the log file is not filled with them.
```